package.json

```diff
@@ -24,9 +24,9 @@
   "homepage": "https://github.com/dashevo/DashJS#readme",
   "dependencies": {
     "@dashevo/dashcore-lib": "^0.18.0",
-    "@dashevo/dapi-client": "^0.7.0",
-    "@dashevo/wallet-lib": "^4.1.1",
-    "@dashevo/dpp": "^0.9.1"
+    "@dashevo/dapi-client": "^0.8.0-dev.14",
+    "@dashevo/wallet-lib": "https://github.com/dashevo/wallet-lib#impr/evonet",
```

Comment:
Will this be merged in the wallet-lib's main branch before this PR is merged? Would be nice to not have a dependency on a branch that will be deleted.

Reply:
Thanks for picking that up ! It should not. Fixing it.

Reply:
Fixed